package.json

```diff
@@ -10,6 +10,7 @@
     "eject": "react-scripts-ts eject",
     "use-prod-config": "cp src/data/prod.config.json src/data/config.json",
     "use-test-config": "cp src/data/test.config.json src/data/config.json",
+    "use-local-config": "cp src/data/local.config.json src/data/config.json",
```

Comment:
Cool, though a little misleading since firebase will never be locally runnable right? lol. w/e